tests/testthat/test-epidatacall.R

```diff
@@ -51,10 +40,14 @@ test_that("fetch_tbl warns on non-success", {
     readRDS(testthat::test_path("data/test-classic.rds")) %>%
     jsonlite::fromJSON() %>%
     `[[<-`("message", "* This is a warning with a leading asterisk and {braces} to make sure we don't have bulleting/glue bugs.")
```

Comment:
```suggestion
    `[[<-`("message", artificial_warning)
```

I introduced `artificial_warning` and must have forgotten to complete the de-dupe.